content/2018-munich/index.md

```diff
@@ -18,13 +18,29 @@ Get an impression of last year's PromCon 2017:
 
 <%= youtube_player("4Pr-z8-r1eo") %>
 
-## Registration
+## Our sponsors
 
-Registration is open.
+A big thank you to all of our sponsors who helped make PromCon 2017 happen!
 
-## Sponsorship
+<h3>Diamond</h3>
```

Comment:
I chose based on how awkward or good the logos looked in a given order, since they have slightly different dimensions. But in order of commitment also looks ok enough, changed to that.